_layouts/curriculum.html

```diff
@@ -66,6 +66,9 @@ <h4 class="modal-title" id="timerModalLabel">Help / Shortcuts</h4>
 </div>
 
 <div class="col-content deck  col-md-12 col-sm-12 col-xs-12 materials curriculum shift-left">
+  <div class="deprecation-notice">
+    Please note: This class is no longer in use. It is now part of our two day </strong><a href="https://training.github.com/kit/">GitHub for Developers course.</a>
```

Comment:
> Please note: This class is no longer in use. It is now part of our two day GitHub for Developers course.

Perhaps a one sentence combo of the above that has a bit more positive wording (i.e. omitting the "no longer..."). Something like this:

**Please note:** This class and curriculum have now become a part of the two day *GitHub for Developers* course.